horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/ChangesServiceImpl.java

```diff
@@ -106,7 +106,7 @@ public List<TimeseriesTarget> query(Query query) {
             sql.append("LEFT JOIN fingerprint fp ON fp.dataset_id = dp.dataset_id WHERE json_equals(fp.fingerprint, (?4)::::jsonb) ");
          }
          sql.append("ORDER BY timestamp ASC");
-         javax.persistence.Query nativeQuery = em.createNativeQuery(sql.toString(), DataPointDAO.class)
+         jakarta.persistence.Query nativeQuery = em.createNativeQuery(sql.toString(), DataPointDAO.class)
```

Comment:
is there any reason for the fully qualified name here? we should import the class